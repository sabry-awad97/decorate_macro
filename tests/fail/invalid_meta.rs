use decorate_macro::decorate;

#[decorate({invalid_option = "test"}, log_execution)]
fn test_function() -> i32 {
    42
}

fn main() {}
