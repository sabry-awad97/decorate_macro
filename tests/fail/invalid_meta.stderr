error: expected identifier
 --> tests/fail/invalid_meta.rs:3:12
  |
3 | #[decorate({invalid_option = "test"}, log_execution)]
  |            ^
